.github/scripts/get-web-socket-api-list.js

```diff
@@ -0,0 +1,64 @@
+const path = require('path');
+const fs = require('fs-extra');
+const YAML = require('yaml');
+
+const BASE_PATH = path.resolve(__dirname, '..', '..');
+const DESTINATION_PATH = path.join(BASE_PATH, 'config/v3/');
+const SOURCE_PATH = path.join(process.argv[2] || 'deriv-websockets-api', 'config/v3/');
+const ALL_CONFIG_PATH = path.join(BASE_PATH, '_data/');
+
+/*
+ *  This function copies all the api schema files from the regentmarkets/binary-websocket-api/config/v3 folder to the binary-com/deriv-api-docs/config/v3 folder,
+ *   whose hidden attribute is set to false.
+ *  It also generates the v3.yml file in the binary-com/deriv-api-docs/_data folder, that lists all the api calls
+ */
+const generateApiList = async () => {
+  console.log('Generating API list...');
```

Comment:
Do we need this console.log?

Reply:
@niloofar-deriv, This is just info that's printed when the action starts copying the files, I can remove it if you want me to.

Reply:
No that would be fine